code/ship/ship.cpp

```diff
@@ -3729,6 +3742,18 @@ static void parse_ship_values(ship_info* sip, const bool is_template, const bool
 			stuff_float(&sip->afterburner_recover_rate);
 		}
 
+		if (optional_string("+Aburn Min Fuel to Start:")) {
```

Comment:
All .tbl additions must be added to the wiki upon merging.  Please be sure to add it.